codelab/codelab.go

```diff
@@ -16,8 +16,10 @@ import (
 
 	"github.com/google/cel-go/cel"
 	"github.com/google/cel-go/common/types"
+	// "github.com/google/cel-go/checker/decls"
```

Comment:
Instead of commenting out the import, you can put an underscore in front of the path to say "keep this, but ignore it for compilation purposes":

```go
import(
    _ "github.com/google/cel-go/checker/decls"
)